doc/Core.xml

```diff
@@ -5369,6 +5394,384 @@ onvif://www.onvif.org/name/ARV-453
+        <variablelist role="op">
+          <varlistentry>
+            <term>request</term>
+            <listitem>
+              <para role="text">This is an empty message.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>response</term>
+            <listitem>
+              <para role="param">Enabled [xs:boolean]</para>
+              <para role="text">Indicates if the authentication failure warning is enabled or disabled</para>
+            </listitem>
+            <listitem>
+              <para role="param">MonitorPeriod [xs:integer]</para>
+              <para role="text">The amount of seconds that will be monitored after the first authentication failure</para>
+            </listitem>
+            <listitem>
+              <para role="param">MaxAuthFailures [xs:integer]</para>
+              <para role="text">Maximum number allowed of authentication failures before reporting the intrusion attempt</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+              <para role="text">No command specific faults defined</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>access class</term>
+            <listitem>
+              <para role="access">READ_SYSTEM</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </section>
+      
+      <section>
+        <title>SetAuthFailureWarningConfiguration</title>
+        <para>A device signaling support for the authentication failure warning via the AuthFailureWarningSupported capability 
+        shall support this operation to set the authentication failure warning configuration.</para>
+        <variablelist role="op">
+          <varlistentry>
+            <term>request</term>
+            <listitem>
+              <para role="param">Enabled [xs:boolean]</para>
+              <para role="text">Used to enable/disable the authentication failure warning</para>
+            </listitem>
+            <listitem>
+              <para role="param">MonitorPeriod [xs:integer]</para>
+              <para role="text">The amount of seconds that will be monitored after the first authentication failure</para>
+            </listitem>
+            <listitem>
+              <para role="param">MaxAuthFailures [xs:integer]</para>
+              <para role="text">Maximum number allowed of authentication failures before reporting the intrusion attempt</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>response</term>
+            <listitem>
+              <para role="text">This is an empty message.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+              <para role="param">env:Sender - ter:InvalidArgVal - ter:ServiceNotSupported</para>
```

Comment:
ter:ServiceNotSupported doesn't seem right here

Reply:
Same as before, we could use the generic "InvalidArgVal, Argument Value Invalid" to signal that the configuration settings are not correct